docs/references/configuration.md

```diff
@@ -0,0 +1,192 @@
+---
+description: Parameter reference
+---
+
+# Configuration
+
+When installing Opstrace, one has to provide a corresponding configuration document in YAML format.
+A minimal variant might look like this:
+
+```yaml
+tenants:
+    - prod
+    - ci
+```
+
+The following sections list individual configuration parameters and aim to provide a detailed specification for their purpose and meaning.
+
+At a high level, there are two different types of parameters:
+
+* required ones
+* optional ones (but please do not assume sane defaults yet :-))
+
+## Required parameters
+
+### `tenants`
+
+The set of initial tenants \(their names\) to create.
+At least one name is required.
+
+*Value type:* list of strings
+
+*Example:*
+
+```yaml
+tenants:
+    - prod
+    - ci
+```
+
```

Comment:
Note: here, let's say something like "note: if you wish to add tenants later, ..." and link to the "Adding tenants" guide :)

Reply:
Great idea.